cinema/serializer.py

```diff
@@ -0,0 +1,18 @@
+from rest_framework import serializers
+from cinema.models import Movie
+
+
+class MovieSerializer(serializers.Serializer):
```

Comment:
Consider using `serializers.ModelSerializer` instead of `serializers.Serializer` for the `MovieSerializer`. This would automatically generate the `create` and `update` methods and map the fields directly from the `Movie` model, reducing boilerplate code.